iothub/device/tests/PlugAndPlay/PnpHelperTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Microsoft.Azure.Devices.Client.PlugAndPlay;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Devices.Client.Tests.PlugAndPlay
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class PnpHelperTests
```

Comment:
xml comments for class

Reply:
for unit test classes, why?

Reply:
How about "Ensures the PnP convention helpers produce content to spec and using different data types"